### ISSUE TITLE
Install Open3D v0.11.1 on Jetson Nano gives "GLFW Error: GLX: No GLXFBConfigs returned" 

### ISSUE THREAD

**Issue by IvanCobotic** [intel-isl/Open3D#2549]
**IMPORTANT: Please use the following template to report the bug.**
_____

**Describe the bug**
I built Open3D from source and installed on a jetson nano. The build and installation seem to work well, running regular Open3D commands is working, but when i try to use the visualizer i get the following error

[Open3D ERROR] GLFW Error: GLX: No GLXFBConfigs returned

I tried running both via ssh and with monitor directly connected to the jetson, both options give the same result.

I also tried the Open3DViewer app, invoked from Open3D/build with  
./bin/Open3D/Open3D
the GUI starts and I can load an STL file, the graphics look great and everything works.

So graphics are working nicely in GUI app, but o3d.visualization.draw_geometries([..]) fails when invoked from a python script.
I´m guessing this indicates i missed some flag or dependency during the build.

any pointers on how to adress this issue would be greatly appreciated.

**To Reproduce**
1. download nano 4.4.1 image at https://developer.nvidia.com/embedded/jetpack
2. flash image to sd card and boot up in jetson nano -> this gives the nvidia jetpack 4.4.1 version of ubuntu 18
3.
```
sudo apt remove --purge cmake
hash -r
sudo snap install cmake --classic
sudo update-alternatives --install /usr/bin/python python /usr/bin/python3.6 1
sudo apt-get -y install python3-pip
sudo update-alternatives --install /usr/bin/pip pip /usr/bin/pip3 1
mkdir -p ~/install && cd ~/install
git clone --recursive https://github.com/intel-isl/Open3D
cd Open3D
#checkout tags/v.0.11.1
git checkout 31fdcadbe4149e833df86612e718c23868c5b72f

sudo apt-get update -y
sudo apt-get install -y apt-utils build-essential 
sudo apt-get install -y python3-dev
sudo apt-get install -y xorg-dev libglu1-mesa-dev
sudo apt-get install -y libblas-dev liblapack-dev liblapacke-dev
sudo apt-get install -y libsdl2-dev libc++-7-dev libc++abi-7-dev libxi-dev
sudo apt-get install -y clang-7

mkdir build
cd build

export CUDA_PATH=/usr/local/cuda-10.2/
export CUDA_INC_PATH=/usr/local/cuda-10.2/include
cmake \
    -DCMAKE_CUDA_COMPILER=/usr/local/cuda-10.2/bin/nvcc \
    -DCMAKE_BUILD_TYPE=Release \
    -DBUILD_SHARED_LIBS=ON \
    -DBUILD_CUDA_MODULE=ON \
    -DBUILD_GUI=ON \
    -DBUILD_TENSORFLOW_OPS=OFF \
    -DBUILD_PYTORCH_OPS=OFF \
    -DBUILD_UNIT_TESTS=ON \
    -DCMAKE_INSTALL_PREFIX=~/open3d_install \
    -DPYTHON_EXECUTABLE=$(which python) \
    ..

make -j$(nproc)
make tests -j$(nproc)
./bin/tests --gtest_filter="-*Reduce*Sum*"

pip install Cython
sudo apt-get install -y libjpeg-dev
make install-pip-package -j$(nproc)
python -c "import open3d; print(open3d)"`


**CMAKE Output**
the cmake output look like this,
-- THIRD_PARTY_DOWNLOAD_DIR is not specified, will download directly.
-- Open3D 0.11.1+31fdcadb
CMake Warning at CMakeLists.txt:180 (message):
  ARM CPU detected, setting USE_BLAS=ON.


CMake Warning at CMakeLists.txt:184 (message):
  ARM CPU detected, setting BUILD_FILAMENT_FROM_SOURCE=ON.


-- Using python from PYTHON_EXECUTABLE variable: /usr/bin/python
-- (PYTHON_EXECUTABLE matches python from PATH)
-- CMAKE_CUDA_COMPILER: /usr/local/cuda-10.2/bin/nvcc
-- CMAKE_CUDA_COMPILER_VERSION: 
-- Building CUDA enabled
-- Autodetected CUDA architecture(s):  5.3
-- CUDA_GENCODES: -gencode arch=compute_53,code=sm_53
-- CMAKE_CUDA_TOOLKIT_INCLUDE_DIRECTORIES: /usr/local/cuda-10.2/targets/aarch64-linux/include
-- CMAKE_CUDA_IMPLICIT_INCLUDE_DIRECTORIES: /usr/include/c++/7;/usr/include/aarch64-linux-gnu/c++/7;/usr/include/c++/7/backward;/usr/lib/gcc/aarch64-linux-gnu/7/include;/usr/local/include;/usr/lib/gcc/aarch64-linux-gnu/7/include-fixed;/usr/include/aarch64-linux-gnu;/usr/include
-- Building library Assimp from source
-- Building with OpenMP
-- Building library 3rdparty_eigen3 from source
-- Building library 3rdparty_flann from source
-- Building library 3rdparty_nanoflann from source
-- Building library 3rdparty_glew from source
-- Building library 3rdparty_glfw3 from source
-- Found Vulkan: /usr/lib/aarch64-linux-gnu/libvulkan.so  
-- Using X11 for window creation
-- Building third-party library JPEG from source
-- NASM assembler not found - libjpeg-turbo performance may suffer
-- libturbojpeg: WITH_CRT_DLL=OFF
-- Building library 3rdparty_jsoncpp from source
-- Building library 3rdparty_lzf from source
-- Building library 3rdparty_tritriintersect from source
-- Building third-party library zlib from source
-- Building third-party library libpng from source
-- Building library 3rdparty_rply from source
-- Building library 3rdparty_tinyfiledialogs from source
-- Building library 3rdparty_tinygltf from source
-- Building library 3rdparty_tinyobjloader from source
-- Building library 3rdparty_qhull_r from source
-- Building library 3rdparty_qhullcpp from source
-- Building library 3rdparty_fmt from source
-- pybind11 v2.5.0
-- Building library 3rdparty_poisson from source
-- Building library 3rdparty_googletest from source
-- Building library 3rdparty_imgui from source
-- Building third-party library Filament from source
-- Filament can only be built with Clang >= 7
-- Using /usr/bin/clang++-7 to build Filament
-- CLANG_LIBDIR: /usr/lib/llvm-7/lib
-- CPP_LIBRARY: /usr/lib/llvm-7/lib/libc++.so
-- CPPABI_LIBRARY: /usr/lib/llvm-7/lib/libc++abi.so
-- A library with LAPACK API found.
-- A cache variable, namely LAPACKE_DIR, has been set to specify the install directory of LAPACKE
-- Looking for LAPACKE_dgeqrf
-- Looking for LAPACKE_dgeqrf - not found
-- Looking for lapacke : test with lapack fails
-- Looking for LAPACKE_dgeqrf
-- Looking for LAPACKE_dgeqrf - found
-- Using system BLAS/LAPACK
-- BUILD_JUPYTER_EXTENSION is set to OFF
-- Using Python version: 3.6
-- Using Python version major: 3
-- PYPI_PACKAGE_NAME: open3d
-- ========================================================================
-- Open3D 0.11.1 Configuration Summary
-- ========================================================================
-- Enabled Features:
--   OpenMP .................................. ON
--   Headless Rendering ...................... OFF
--   Azure Kinect Support .................... OFF
--   CUDA Support ............................ ON
--   Build GUI ............................... ON
--   Build Shared Library .................... ON
--   Build Unit Tests ........................ ON
--   Build Examples .......................... 
--   Build Python Module ..................... ON
--   - with Jupyter Notebook Support ......... OFF
--   Build TensorFlow Ops .................... OFF
--   Build PyTorch Ops ....................... OFF
--   Build Benchmarks ........................ OFF
--   Bundle Open3D-ML ........................ OFF
--   Build RPC interface ..................... OFF
--   Force GLIBCXX_USE_CXX11_ABI= ............ 0
-- ========================================================================
-- Third-Party Dependencies:
--   EIGEN3 .................................. yes (build from source)
--   FILAMENT ................................ yes (build from source)
--   FLANN ................................... yes (build from source)
--   FMT ..................................... yes (build from source)
--   GLEW .................................... yes (build from source)
--   GLFW .................................... yes (build from source)
--   GOOGLETEST .............................. yes (build from source)
--   IMGUI ................................... yes (build from source)
--   JPEG .................................... yes (build from source)
--   JSONCPP ................................. yes (build from source)
--   LIBLZF .................................. yes (build from source)
--   OPENGL .................................. yes (build from source)
--   PNG ..................................... yes (build from source)
--   PYBIND11 ................................ yes (build from source)
--   QHULL ................................... yes (build from source)
--   LIBREALSENSE ............................ no
--   TINYFILEDIALOGS ......................... yes (build from source)
--   TINYGLTF ................................ yes (build from source)
--   TINYOBJLOADER ........................... yes (build from source)
-- ========================================================================
-- Configuring done
-- Generating done
-- Build files have been written to: /home/ivan/install/Open3D/build

**Environment (please complete the following information):**

- Operating system: Ubuntu 18.04, nvidia jetpack (nano) 4.4.1
- Python version: Python 3.6
- Open3D version: v0.11.1
- Is this remote workstation?: no
- How did you install Open3D?: build from source
- Compiler version (if built from source): see cmake output
```

**Comment by rjsgml5698** [comment]
I know open3D as OpenGL 4.1, are you satisfied with this?

**Comment by yxlao** [comment]
@IvanCobotic we're working on a new `draw()` to replace `draw_geometries()` in https://github.com/intel-isl/Open3D/pull/2585, where `draw` launches a stripped-down version of the GUI app.

So if the GUI app works for you then it's likely the `draw` function will also work.

**Comment by IvanCobotic** [comment]
Hello,
Thanks for the feedback. We did some more tracking on this issue and
ultimately figured out that the OpenGL does not work with external DISPLAY
in the latest Nvidia Jetpack for jetson (jetpack 4.4.1). I verified that in
jetpack 4.3 the openGL DISPLAY was working but in jetpack 4.4 it got
broken, causing the error I reported to you.

So this issue is not Open3D specific, it's a driver problem of Nvidias.

Best regards

Ivan Lundberg

CEO Cobotic AB

+46700908820
